numbat/modules/math/combinatorics.nbt

```diff
@@ -0,0 +1,34 @@
+use core::error
+use core::functions
+use math::transcendental
+
+@name("Factorial")
+@description("The product of the integers 1 through n, also written n!")
```

Comment:
You can use `$…$` to embed LaTeX formulas in `@description`s. This way, they render more nicely in the online documentation.

Reply:
Maybe this was your intent, but I think we could state more clearly that Numbat also supports `!` as an operator for factorials.